system/services/websiteUsers/WebsiteLoginService.cfc

```diff
@@ -569,6 +569,17 @@ component displayName="Website login service" {
 		return false;
 	}
 
+	public void function setUserSessionById( required string id ) {
```

Comment:
This feels somewhat dangerous. If someone is able to edit another user, then will they not then become logged in as that user with this change?

I think it may be preferable to implement specific session refresh in any implementations that perform editing of self through the frontend rather than globally when updating any user record.

Reply:
Replace the interceptor with helper.